.github/workflows/go.yml

```diff
@@ -7,73 +7,66 @@ on:
     branches: "*"
 
 jobs:
-
   build:
     name: ${{ matrix.os }} - Go ${{ matrix.go_version }}
     runs-on: ubuntu-latest
     strategy:
-     # If you want to matrix build , you can append the following list.
+      # If you want to matrix build , you can append the following list.
       matrix:
         go_version:
-          - 1.13
-          - 1.14
+          - 1.18
+          - 1.19
         os:
           - ubuntu-latest
 
     steps:
+      - name: Set up Go ${{ matrix.go_version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go_version }}
+        id: go
 
-    - name: Set up Go ${{ matrix.go_version }}
-      uses: actions/setup-go@v2
-      with:
-        go-version: ${{ matrix.go_version }}
-      id: go
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-
-    - name: Cache build dependence
-      uses: actions/cache@v2
-      with:
-        # Cache
-        path: ~/go/pkg/mod
-        # Cache key
-        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-        # An ordered list of keys to use for restoring the cache if no cache hit occurred for key
-        restore-keys: ${{ runner.os }}-go-
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
 
-    - name: Install goimports
-      run: go get golang.org/x/tools/cmd/goimports
```

Comment:
goimports check is necessary to format code

Reply:
It's recommend to use linters in golangci-lint.